model/connection_manager.rb

```diff
@@ -19,28 +19,24 @@ def call(env)
   protected
 
   def open_connections_if_necessary!
-    self.class.connection ||= begin
+    self.class.db ||= begin
       add_finalizer_hook!
-      {
-        :users => open_connection(:users),
-        :forms => open_connection(:forms),
-        :form_data => open_connection(:form_data),
-      }
+      Sequel.connect(
+        :adapter => 'postgres',
+        :host => ConfigManager['database']['host'],
+        :database => ConfigManager['database']['database'],
+        :user => ConfigManager['database']['user'],
+        :password => ConfigManager['database']['password']
+      )
     end
   end
 
-  def open_connection(db_name)
-    TokyoTyrant.const_get(ConfigManager['database'][db_name.to_s]['type']).new(
-      ConfigManager['database'][db_name.to_s]['host'],
-      ConfigManager['database'][db_name.to_s]['port'])
-  end
-
   def add_finalizer_hook!
     at_exit do
       begin
-        self.class.connection.each_value{ |connection| connection.close }
+        self.class.db.disconnect
       rescue Exception => e
-        puts "Error closing Tokyo Cabinet connection. You might have to clean up manually."
+        puts "Error closing Postgres connection. Some minor data loss may have occurred for in-flight transactions."
```

Comment:
This is not very friendly to non-technical users who will think about ✈️. Why not just "ongoing transfers" or something like that?